src/test/java/de/unistuttgart/iste/gits/content_service/ContentServiceApiTests.java

```diff
@@ -15,10 +19,14 @@
 
 @SpringBootTest(classes = ContentServiceApplication.class)
 @AutoConfigureGraphQlTester
+@ActiveProfiles("test")
 class ContentServiceApiTests {
     @Autowired
     private GraphQlTester graphQlTester;
 
+    @ClassRule
+    public static PostgreSQLContainer<GitsPostgresSqlContainer> postgreSQLContainer = GitsPostgresSqlContainer.getInstance();
```

Comment:
@ClassRule is a Junit4 concept and were using Junit 5. Does the library not support Junit 5? Should usually be done with @ExtendWith

Reply:
Okay I think it should be done with @Container, see https://java.testcontainers.org/test_framework_integration/junit_5/